### ISSUE TITLE
Hyperparameter Search

### ISSUE THREAD

**Issue by tomiock** [tomiock/TARS#17]
Use sweeps from `wandb`. Target metric: **accuracy**.

Changes:
- The classics: learning rate, layers, ....
- metric learning stuff: distance for the miner, equivalent losses, margin, etc...
